service-tests/uptimerobot.js

```diff
@@ -58,22 +59,22 @@ t.create('Uptime Robot: Status (unexpected response, invalid json)')
   .get('/status/m778918918-3e92c097147760ee39d02d36.json')
   .intercept(nock => nock('https://api.uptimerobot.com')
     .post('/v2/getMonitors')
-    .reply(200, "{{{{{invalid json}}")
+    .reply(invalidJSON)
   )
   .expectJSON({name: 'status', value: 'inaccessible'});
 
 t.create('Uptime Robot: Percentage (valid)')
   .get('/ratio/m778918918-3e92c097147760ee39d02d36.json')
   .expectJSONTypes(Joi.object().keys({
     name: 'uptime',
-    value: isDecimalPercentage,
+    value: isPercentage,
   }));
 
 t.create('Uptime Robot: Percentage (valid, with numberOfDays param)')
   .get('/ratio/7/m778918918-3e92c097147760ee39d02d36.json')
   .expectJSONTypes(Joi.object().keys({
     name: 'uptime',
-    value: isDecimalPercentage,
+    value: isPercentage,
```

Comment:
Currently API returns `custom_uptime_ratio` equal `100.000` which is changed to `100` in 
https://github.com/badges/shields/blob/8ad176d014bdddc95f06a95ae4ff82c7697bb569/server.js#L7370 And 100 is not a decimal value. That's why created `isPercentage`.